pyotgw/pyotgw.py

```diff
@@ -358,53 +358,84 @@ async def get_status(self):
+                v.DATA_DHW_BURNER_HOURS: int(fields[33]),
+            }
+        else:
+            capmodlimits = fields[4].split("/")
+            dhw_setp_bounds = fields[13].split("/")
+            ch_setp_bounds = fields[14].split("/")
+            status = {
+                v.DATA_SLAVE_MAX_RELATIVE_MOD: float(fields[3]),
+                v.DATA_ROOM_SETPOINT: float(fields[5]),
+                v.DATA_REL_MOD_LEVEL: float(fields[6]),
+                v.DATA_CH_WATER_PRESS: float(fields[7]),
+                v.DATA_ROOM_TEMP: float(fields[8]),
+                v.DATA_CH_WATER_TEMP: float(fields[9]),
+                v.DATA_DHW_TEMP: float(fields[10]),
+                v.DATA_OUTSIDE_TEMP: float(fields[11]),
+                v.DATA_RETURN_WATER_TEMP: float(fields[12]),
+                v.DATA_DHW_SETPOINT: float(fields[15]),
+                v.DATA_MAX_CH_SETPOINT: float(fields[16]),
+                v.DATA_TOTAL_BURNER_STARTS: int(fields[17]),
+                v.DATA_CH_PUMP_STARTS: int(fields[18]),
+                v.DATA_DHW_PUMP_STARTS: int(fields[19]),
+                v.DATA_DHW_BURNER_STARTS: int(fields[20]),
+                v.DATA_TOTAL_BURNER_HOURS: int(fields[21]),
+                v.DATA_CH_PUMP_HOURS: int(fields[22]),
+                v.DATA_DHW_PUMP_HOURS: int(fields[23]),
+                v.DATA_DHW_BURNER_HOURS: int(fields[24]),
+            }
+        status.update(
+            {
+                v.DATA_MASTER_CH_ENABLED: int(master_status[7]),
+                v.DATA_MASTER_DHW_ENABLED: int(master_status[6]),
+                v.DATA_MASTER_COOLING_ENABLED: int(master_status[5]),
+                v.DATA_MASTER_OTC_ENABLED: int(master_status[4]),
+                v.DATA_MASTER_CH2_ENABLED: int(master_status[3]),
+                v.DATA_SLAVE_FAULT_IND: int(slave_status[7]),
+                v.DATA_SLAVE_CH_ACTIVE: int(slave_status[6]),
+                v.DATA_SLAVE_DHW_ACTIVE: int(slave_status[5]),
+                v.DATA_SLAVE_FLAME_ON: int(slave_status[4]),
+                v.DATA_SLAVE_COOLING_ACTIVE: int(slave_status[3]),
+                v.DATA_SLAVE_CH2_ACTIVE: int(slave_status[2]),
+                v.DATA_SLAVE_DIAG_IND: int(slave_status[1]),
+                v.DATA_CONTROL_SETPOINT: float(fields[1]),
+                v.DATA_REMOTE_TRANSFER_DHW: int(remote_params[0][7]),
+                v.DATA_REMOTE_TRANSFER_MAX_CH: int(remote_params[0][6]),
+                v.DATA_REMOTE_RW_DHW: int(remote_params[1][7]),
+                v.DATA_REMOTE_RW_MAX_CH: int(remote_params[1][6]),
+                v.DATA_SLAVE_MAX_CAPACITY: int(capmodlimits[0]),
+                v.DATA_SLAVE_MIN_MOD_LEVEL: int(capmodlimits[1]),
+                v.DATA_SLAVE_DHW_MAX_SETP: int(dhw_setp_bounds[0]),
+                v.DATA_SLAVE_DHW_MIN_SETP: int(dhw_setp_bounds[1]),
+                v.DATA_SLAVE_CH_MAX_SETP: int(ch_setp_bounds[0]),
+                v.DATA_SLAVE_CH_MIN_SETP: int(ch_setp_bounds[1]),
+            }
```

Comment:
We're still missing the following fields for v5: 3, 4, 17, 18, 23, 24, 25.

Reply:
I added the fields